src/requests/init.jl

```diff
@@ -24,7 +24,7 @@ const serverCapabilities = ServerCapabilities(
     true,
     false,
     ExecuteCommandOptions(missing, collect(keys(LSActions))),
-    false,
+    true,
```

Comment:
This is `selectionRangeProvider`, right? Did we always support that? If so, why was it disabled?

Reply:
I must have been fiddling with something, will turn off